src/Libraries/PythonNodeModelsWpf/PythonNodeModelsWpf.csproj

```diff
@@ -100,6 +100,7 @@
 	<ProjectReference Include="..\DSIronPython\DSIronPython.csproj">
       <Project>{9eef4f42-6b3b-4358-9a8a-c2701539a822}</Project>
       <Name>DSIronPython</Name>
+      <Private>False</Private>
```

Comment:
This will not copy the DSIronPython.dll into the nodes folder as we have that binary in the debug folder. Before this, it was being copied into 2 locations.